### DISCUSSION TITLE
Invalid in fieldState with useController v7 deprecated

### DISCUSSION THREAD

**Discussion by vietvh-3042** [react-hook-form/react-hook-form#8983] [Q&A] [answered]
I using invalid in fieldState with useController but in version v7 it deprecated. I don't know if there is any change in this case

**Comment by Moshyfawn** [comment, marked as answer]
It's getting deprecated because it's kind of redundant since you can do this instead:
```js
!!fieldState.error
```

**Reply by vietvh-3042** [reply]
Ok, got it! it's all work now, thanks!
